src/scripts.js

```diff
@@ -70,8 +77,15 @@ function updateMainPageRecipeIcons() {
   icon2Img.src = allRecipes[getRandomIndex(allRecipes)].image;
   icon3Img.src = allRecipes[getRandomIndex(allRecipes)].image;
   icon4Img.src = allRecipes[getRandomIndex(allRecipes)].image;
+  icon5Img.src = allRecipes[getRandomIndex(allRecipes)].image;
+  icon6Img.src = allRecipes[getRandomIndex(allRecipes)].image;
 }
 
+function updateMainPaigeFeatureImg(){
```

Comment:
There is a small typo in the spelling of this function. "page" has an "i" in it

Reply:
typo fixed, thanks for catching this!